modules/ROOT/pages/sap/sap-s4hana-cloud-connector-create-batch.adoc

```diff
@@ -0,0 +1,40 @@
+= Batch Processing
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The batch processing feature is available starting in SAP S/4HANA OData Cloud Connector version 2.1. 
+
+You must follow the correct order for structural batch operations:
+ 
+. The batch must be initiated before any requests are added into it.
+. Requests cannot be added to the batch or the change set after it is executed.
+
+== Create a Batch Operation
+
+To create and process a batch operation, you must:
+
+. Place the *Initiate Batch* operation in the Mule flow, which returns a unique batch identifier. +
+This unique batch identifier should be stored, for example, in the flow variable, so that it can then be used to group the batch data operations under the common batch request.
+. Use the *Create Change Set* operation to create a new change set. +
+This operation returns a unique change set identifier, which must then be used in the *Batch Create*, *Batch Update*, *Delete Entity*, or *Batch Execute Function* operations in order to add them to the change set.
++
+The *Batch Create Entity*, *Batch Get Entity*, *Batch Query*, and *Batch Execute Function* operations return a JSON result, however, the result of these operations cannot be read until the *Execute Batch* operation is executed. 
++
+If you attempt to read the result before the *Execute Batch* operation is executed, the connector throws an exception. You can store the output of these operations, for instance, into a flow variable, and access the output after the *Execute Batch* operation is executed.
++
+. When the batch is complete, use the *Execute Batch* operation to send a batch request.
+. After the *Execute Batch* operation is successfully executed, a summary is returned, which contains basic information about the outcome of every batch operation that was included as part of the batch operation. The detailed results of these batch operations are then available to read.
+
+== Troubleshooting
+
+Exceptions are thrown in the following circumstances:
+
+* The connector throws an exception if an operation references an invalid batch identifier, either because it was never initiated or if the *Execute Batch* operation was already performed for that particular batch identifier. 
```

Comment:
```suggestion
* An operation references a batch identifier that is invalid, either because:
** It was never initiated.
** It was already processed as part of an *Execute Batch* operation. 
```